.rubocop.yml

```diff
@@ -791,6 +801,9 @@ Rails/TimeZone:
     - strict
     - flexible
 
+Rails/WhereNot:
+  Enabled: false
```

Comment:
`.where.not(x: nil)` is not always intuitive so again I want to trust our devs on this one